### DISCUSSION TITLE
Turn off Autoplay for video links

### DISCUSSION THREAD

**Discussion by OlafKwakman** [mozilla/hubs#4728] [Spoke Questions] [answered]
We have created an experience centre with Hubs, with a lot of videos.
To limit the file size, we have used YouTube video links instead of uploading all videos.
But every time we enter the sopace, all the videos start playing.

Is there a way to turn off autoplay for video links in Mozilla Hubs?

**Comment by pattersonbl2** [comment, marked as answer]
are you adding the links in spoke ?  if so, you should see this to disable autoplay
![image](https://user-images.githubusercontent.com/28871018/142906539-18983420-a276-462d-aff3-39a0de6a84dc.png)
